### ISSUE TITLE
Issues with expand_uvm_verbosity settings in common_sim_cfg.hjson

### ISSUE THREAD

**Issue by svenka3** [lowRISC/opentitan#4399]
In common_sim_cfg.hjson I see *expand_uvm_verbosity_* settings. 2 issues I see:

1. Missing UVM_FULL (I added below for my own use)
2. Why UVM_DEBUG? - this is strictly for UVM Lib developers, not meant for users. 

```json
 expand_uvm_verbosity_n: UVM_NONE
  expand_uvm_verbosity_l: UVM_LOW
  expand_uvm_verbosity_m: UVM_MEDIUM
  expand_uvm_verbosity_h: UVM_HIGH
  expand_uvm_verbosity_f: UVM_FULL
  expand_uvm_verbosity_d: UVM_DEBUG
```

**Comment by rswarbrick** [comment]
Yep, that doesn't sound quite right to me. @sriyerg would you prefer to change the names of the dvsim arguments or just change the last line in that example to `expand_uvm_verbosity_d: UVM_FULL` ?

**Comment by sriyerg** [comment]
We have never had the need to set UVM_FULL. If you want to see UVM phase transitions, you can set +UVM_PHASE_TRACE plusarg. Though, let us know under what circumstances is it useful. 

It causes unnecessary confusion during debug time, which verbosity level to enable. So we want to encourage the users to limit to low (critical 'startup' type messages), medium (transaction flow messages) and high (uvm testbench debug, detailed messages from forever loops). You are right about UVM_DEBUG - we should cleanup our codebase and remove all instances of it, since it is reserved for debugging the UVM codebase.

**Comment by sriyerg** [comment]
Actually, it needs to remain in the HJSon so that users can still set it via the command line switch `-v d`.

**Comment by svenka3** [comment]
With due respect @sriyerg - why take away a UVM feature (UVM_FULL verb in this case) from your users? Imagine someone adding a new test, wanting to add a message with UVM_FULL verbosity (Just for making a case here). Also, once you decide to clean-up UVM_DEBUG (I am not sure how many places it is used in OT codebase), what will you move them to? UVM_FULL maybe?

So my suggestion - please add

  expand_uvm_verbosity_f: UVM_FULL

One line won't harm much :-)

Cheers

**Comment by sriyerg** [comment]
IMO 4 verbosity levels is an overkill. Can you justify a scenario where the three most used ones - low, medium and high are not enough?

**Comment by svenka3** [comment]
https://github.com/search?q=uvm_full&type=code

Just on GitHub some 1000 hits. Imagine how many users across the globe are used to this "feature"

I will rest the case here, upto you folks

**Comment by sriyerg** [comment]
Fixed with #4429.
